dist/ads.js

```diff
@@ -127,14 +132,26 @@ const parseSellersJSON = async (response) => {
             seller_count: 0,
           }
         },
-        passthrough_count: 0
+        passthrough_count: 0,
+        confidential_count: 0
       }
     };
 
     // Clean up file content
     result.seller_count = content.sellers.length;
 
     for (let seller of content.sellers) {
+      // Validating records
+      if (!seller.seller_type && !seller.domain) {
+        continue;
+      }
+      if (!seller.seller_type) {
+        seller.seller_type = "";
+      }
+      if (!seller.domain) {
+        seller.domain = "";
+      }
```

Comment:
If no domain - it's either confidential seller or non-web partner.
We may skip it.